packages/powersync/lib/src/database/native/native_powersync_database.dart

```diff
@@ -54,6 +54,8 @@ class PowerSyncDatabaseImpl
   /// Use [attachedLogger] to propagate logs to [Logger.root] for custom logging.
   late final Logger logger;
 
+  Map<String, dynamic>? clientParams;
```

Comment:
I'm not sure why this is public or what it's used for - but I think this should be on `PowerSyncDatabaseMixin` in order for it to be available on all the implementations.

Reply:
Its related to this for adding user parameters support (#98).